src/wx/openal.cpp

```diff
@@ -107,44 +107,46 @@ void OpenAL::debugState()
     ALint value = 0;
     alGetSourcei(source, AL_SOURCE_STATE, &value);
     ASSERT_SUCCESS;
-    winlog(" soundPaused = %i\n", soundPaused);
-    winlog(" Source:\n");
-    winlog("  State: ");
-
-    switch (value) {
-    case AL_INITIAL:
-        winlog("AL_INITIAL\n");
-        break;
-
-    case AL_PLAYING:
-        winlog("AL_PLAYING\n");
-        break;
-
-    case AL_PAUSED:
-        winlog("AL_PAUSED\n");
-        break;
-
-    case AL_STOPPED:
-        winlog("AL_STOPPED\n");
-        break;
-
-    default:
-        winlog("!unknown!\n");
-        break;
-    }
+
+    //winlog(" soundPaused = %i\n", soundPaused);
```

Comment:
If the `winlog()` function is a problem, just redefine it to something else with a macro on linux.

Reply:
`winlog()` was being undefined by a macro and then replaced by `//`. So essentially it was just commenting the code anyway. It felt weird removing the code, since it could be actually being used by debug mode. I need to check if we cant' use something for debug mode.

Reply:
It was perhaps defined to a debug log function by the author of this code in eons past.

This construction should work for our purposes:

```cpp
#define winlog(x,...) do {} while(0)
```

it also produces no warnings with -Wall.

Taken from:

https://stackoverflow.com/a/1306690/262458